charts/argo/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: v2.12.5
 description: A Helm chart for Argo Workflows
 name: argo
-version: 0.15.4
+version: 0.15.5
```

Comment:
You’re adding a “feature” (new value) to the chart, the new version should be 0.16.0 ;-)

Reply:
Bumped to 0.16.0